libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ErrorResponse.java

```diff
@@ -106,7 +106,8 @@ public enum ErrorResponse {
     UNEXPECTED_CREDENTIAL_TYPE(1092, "Unexpected credential type"),
     IPV_SESSION_NOT_FOUND(
             1093, "Invalid IPV session id provided, corresponding session not found in db"),
-    CLIENT_OAUTH_SESSION_NOT_FOUND(1093, "Client OAuth session not found");
+    CLIENT_OAUTH_SESSION_NOT_FOUND(1093, "Client OAuth session not found"),
+    MISSING_LANGUAGE(1093, "Missing language choice from the frontend");
```

Comment:
Not sure what's going on here - I think these error codes should be unique, and probably not re-used even if the old error is no longer used. There 1093 is already being used above for IPV_SESSION_NOT_FOUND so not sure why it appears to also have been used for CLIENT_OAUTH_SESSION_NOT_FOUND

Reply:
Ah that's an error. I'll update it so it increments. I'll add a test for this to ensure the error responses don't overlap - I see the same with 1051